github/event_types.go

```diff
@@ -149,6 +149,12 @@ type EditChange struct {
 	Title *struct {
 		From *string `json:"from,omitempty"`
 	} `json:"title,omitempty"`
+	Name *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"title,omitempty"`
+	Note *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"title,omitempty"`
 	Body *struct {
 		From *string `json:"from,omitempty"`
 	} `json:"body,omitempty"`
```

Comment:
@gmlewis Having longer discussions inside inlined comments that may get collapsed is hard, so I'll quote your comment and reply at the bottom.